recipes/ibm_db_sa/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ibm_db_sa" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aed72b7275ed5327501f179a0302eaf8934e084aeb3c27a7e0b68f1c83f06173
+
+build:
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [linux64]
```

Comment:
Removing this will fail one Linux job but spawn another. It's OK if the initial one is failing.
```suggestion
```

Reply:
Addressed